### ISSUE TITLE
Adding a local box via "box add" fails with v1.8.7, succeeds with v1.7.4

### ISSUE THREAD

**Issue by arcuri82** [mitchellh/vagrant#8003]
Using OS X El Capitan, 10.11.3

With Vagrant 1.8.7, the following command fails with no explanation to help debugging it:

```
Andreas-MBP:~ foo$ vagrant box add --name cdkv2 rhel-cdk-kubernetes-7.2-29.x86_64.vagrant-virtualbox.box 
==> box: Box file was not detected as metadata. Adding it directly...
==> box: Adding box 'cdkv2' (v0) for provider: 
    box: Unpacking necessary files from: file:///Users/foo/rhel-cdk-kubernetes-7.2-29.x86_64.vagrant-virtualbox.box
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.
```

there is a previous bug report for Windows that mentions something similar, and that is now closed:
[https://github.com/mitchellh/vagrant/issues/6725](https://github.com/mitchellh/vagrant/issues/6725)

I tried to downgrade to 1.7.4, and now it works fine:
```
Andreas-MBP:~ foo$ vagrant box add --name cdkv2 rhel-cdk-kubernetes-7.2-29.x86_64.vagrant-virtualbox.box  
==> box: Box file was not detected as metadata. Adding it directly...
==> box: Adding box 'cdkv2' (v0) for provider: 
    box: Unpacking necessary files from: file:///Users/foo/rhel-cdk-kubernetes-7.2-29.x86_64.vagrant-virtualbox.box
==> box: Successfully added box 'cdkv2' (v0) for 'virtualbox'!
```

Note: I was following the instructions from 

https://access.redhat.com/documentation/en/red-hat-container-development-kit/2.2/paged/installation-guide/chapter-4-installing-container-development-kit-on-mac-os-x

which requires to install some Vagrant plugins. Maybe those are the ones screwing Vagrant 1.8.x?

**Comment by debo** [comment]
@arcuri82 did you see this issue? https://github.com/mitchellh/vagrant/issues/7997

**Comment by kikitux** [comment]
Duplicate of https://github.com/mitchellh/vagrant/issues/7997

@arcuri82 There is an issue on macOS in 1.8.7. Fix will be on 1.8.8, you can try 1.8.6 in the mean time.
